pkg/liqoctl/install/validation.go

```diff
@@ -71,7 +71,7 @@ func (o *Options) validateClusterName() (err error) {
 
 	errs := validation.IsDNS1123Label(o.ClusterName)
 	if len(errs) != 0 {
-		return fmt.Errorf("the cluster name may only contain lowercase letters, numbers and hyphens, and must not be no longer than 63 characters")
+		return fmt.Errorf("the cluster name may only contain lowercase letters, numbers and hyphens, and must not be no longer than 63 characters. Try using 'liqoctl --cluster-name' to overcome this error")
```

Comment:
One small suggestion, can we also add some text suggesting that the name should be `DNS compatible`? 

Otherwise this looks great. Thanks for fixing this.